app/main.py

```diff
@@ -1,6 +1,32 @@
+from os import getenv, environ
+
+import requests
+
+API_KEY = getenv("API_KEY")
+API_URL = "https://api.weatherapi.com/v1/current.json"
```

Comment:
The variable `API_URL` should be in uppercase as it is a constant. According to the checklist: 'Use uppercase style for constants' names'.